www/package.json

```diff
@@ -3,6 +3,8 @@
   "description": "Paragon Pattern Library Documentation",
   "version": "1.0.0",
   "dependencies": {
+    "@docsearch/css": "^3.0.0",
+    "@docsearch/react": "^3.0.0",
```

Comment:
[curious/clarification] Hmm, it feels like the 27k line additions for package-lock.json file just by adding these 2 packages might be excessive? I am wondering if the package-lock.json file was possibly deleted and then re-created? For example, when I install these packages, I'm seeing much fewer line additions.

Reply:
This issue has been resolved, please take a look.